### ISSUE TITLE
LocalAuth not working on MacOS

### ISSUE THREAD

**Issue by iamvinny** [pedroslopez/whatsapp-web.js#2431]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When I attempt to use the LocalAuth method for authentication with `whatsapp-web.js`, the script does not function as expected. Without LocalAuth, the script works perfectly fine. After scanning the displayed QR code for authentication with LocalAuth, there is no response, and the expected log message `Client is ready!` is not printed in the terminal.

### Expected behavior

After scanning the QR code, I expect the script to print the log message `Client is ready!` in the terminal and to start listening for messages.

### Steps to Reproduce the Bug or Issue

1. Create a new Node.js project.
2. Install the `whatsapp-web.js` and `qrcode-terminal` packages.
3. Configure the script to use `authStrategy: new LocalAuth()` as shown in the code snippet below:
```js
const { Client, LocalAuth } = require('whatsapp-web.js');

const client = new Client({
    authStrategy: new LocalAuth(),
});
```

4. Execute the script using the command `node index.js`.
5. Observe the display of the QR code.
6. Attempt to scan the QR code with WhatsApp.
7. Note that there is no response, and the log message `Client is ready!` does not appear in the terminal.

### Relevant Code

Below is the full code:

```js
const { Client, LocalAuth } = require('whatsapp-web.js');
const qrcode = require('qrcode-terminal');

const client = new Client({
    authStrategy: new LocalAuth(),
    puppeteer: {
        headless: false,
        args: [
            '--no-sandbox',
            '--disable-setuid-sandbox',
            '--disable-dev-shm-usage'
        ]
    }
});

client.on('qr', qr => {
    qrcode.generate(qr, {small: true});
});

client.on('ready', () => {
    console.log('Client is ready!');
});

client.initialize();

client.on('message', message => {
	if(message.body === '!ping') {
            console.log("Mensagem Recebida: " + message.body);
	    // message.reply('pong'); // reply by quoting the message received
            client.sendMessage(message.from, 'pong'); // Reply without quoting
	}
});
```

### Browser Type

Chromium

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

Yes, I am using Multi Device

### Environment

**OS:** MacOS Sonoma 14.0
**Phone OS:** Android
**whatsapp-web.js:** 1.22.1
**WhatsApp Web Version:** 2.2335.6
**Node.js Version:** v20.5.0

### Additional context
I have also tried deleting all browser data (Cookies, Cache, etc...) and scanning the QR code again, with no success.

Any assistance in resolving this issue would be greatly appreciated. Let me know if there is any additional information or context that would be helpful in addressing this problem.

**Comment by diegofemello** [comment]
same here, on windows

**Comment by RafaelIngenium** [comment]
same here, on windows and linux

**Comment by iamvinny** [comment]
The suggestion posted on [this issue](https://github.com/pedroslopez/whatsapp-web.js/issues/2422#issuecomment-1687313278) provides a temporary fix for the issue until a PR is merged to address the issue.

**Comment by nasruldev** [comment]
Me too. Windows

**Comment by qiraxyz** [comment]
> 

maybe u can change the puppeter headless to headless : "new" or delete the puppeter in client.

**Comment by qiraxyz** [comment]
> > 
> 
> maybe u can change the puppeter headless to headless : "new" or delete the puppeter in client.


```javascript
const { Client, LocalAuth } = require('whatsapp-web.js');
const qrcode = require('qrcode-terminal');

const client = new Client({
    authStrategy: new LocalAuth(),
    puppeteer: {
        headless: "true",
        args: [
            '--no-sandbox',
            '--disable-setuid-sandbox',
            '--disable-dev-shm-usage'
        ]
    }
});

client.on('qr', qr => {
    qrcode.generate(qr, {small: true});
});

client.on('ready', () => {
    console.log('Client is ready!');
});

client.initialize();

client.on('message', message => {
	if(message.body === '!ping') {
            console.log("Mensagem Recebida: " + message.body);
	    // message.reply('pong'); // reply by quoting the message received
            client.sendMessage(message.from, 'pong'); // Reply without quoting
	}
});
```


OR U CAN TRY THIS

```javascript 
const { Client, LocalAuth } = require('whatsapp-web.js');
const qrcode = require('qrcode-terminal');

const client = new Client({
    authStrategy: new LocalAuth(),
});

client.on('qr', qr => {
    qrcode.generate(qr, {small: true});
});

client.on('ready', () => {
    console.log('Client is ready!');
});

client.initialize();

client.on('message', message => {
	if(message.body === '!ping') {
            console.log("Mensagem Recebida: " + message.body);
	    // message.reply('pong'); // reply by quoting the message received
            client.sendMessage(message.from, 'pong'); // Reply without quoting
	}
});
```

**Comment by iamvinny** [comment]
Fixed at abac063b779570729476cf42e29dc694e5345ca6
